### ISSUE TITLE
Emoji avatar is displayed shifted in the QR code screen

### ISSUE THREAD

**Issue by Raiden-GH** [deltachat/deltachat-ios#2200]
When an emoji is used as an avatar, it is displayed shifted upwards in the QR code screen.

DC-iOS 1.46.2

![Emoji-Avatar](https://github.com/deltachat/deltachat-ios/assets/108235546/7698b0eb-c8d3-4c88-825c-40d6633dfd4b)

**Comment by r10s** [comment]
yeah, this comes form the svg renderers that are buggy on different levels across platforms :/

**Comment by Simon-Laux** [comment]
using https://github.com/RazrFalcon/resvg might work better, font alignment in my test is correct, but on my system it has problems loading fonts at all. maybe the wasm version in a web view would work better / more consistently. the advantage to rest is that they aim to be very standard compliant and have a huge test suite.

Also macOS Finder displays the svg just fine for me, so maybe the system can already do it without extra libraries on newer phones? (though I don't know if that good svg support is native or a quicklook extension of one of my image editors)

**Comment by r10s** [comment]
well, switching to another library might come with other issues, than only misplaced emoji. this was also already stated above, "problems loading fonts at all". 

cross-platform svg with text is fundamentally broken :)

ftr, it is might be that we shift to the new "raw" qr code api that is already in use for proxy ([`dc_create_qr_svg()`](https://c.delta.chat/classdc__context__t.html#a46700da7e68f9550a72355680aa3c9d9)), it would be up to the system to add avatar, text whatever - it is not totally decided, there is no urgency, and we'd prototype this on android first, however, closing this issue as **won't fix** to avoid more time being invested here
